app/routes/admin/content/events.js

```diff
@@ -6,12 +6,12 @@ export default Route.extend({
   },
   async model() {
     return {
-      'eventTopics': await this.get('store').query('event-topic', {
+      'eventTopics': await this.get('store').findAll('event-topic', {
```

Comment:
@pradeepgangwar Will we have to change it at other places as well?

Reply:
You cannot use `findAll` here. `findAll` will not support query parameters. You _must_ use `query`

Reply:
Reload the model if need-be but don't use findAll when you need to query stuff.